backend/tarsy/services/dashboard_broadcaster.py

```
@@ -172,6 +172,52 @@ async def _get_and_clear_buffer(self, channel: str) -> List[Dict[str, Any]]:
+        buffered_messages = await self._get_and_clear_buffer(channel)
+        if not buffered_messages:
+            logger.debug(f"No buffered messages for {channel}")
+            return 0
+        
+        logger.info(f"Flushing {len(buffered_messages)} buffered messages for {channel} (triggered by subscription)")
+        
+        sent_count = 0
+        for buffered_msg in buffered_messages:
+            for user_id in subscribers:
+                if not self._should_throttle_user(user_id, channel):
+                    if await self.connection_manager.send_to_user(user_id, buffered_msg):
```

Ruff (0.13.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)